install_config/registry/securing_and_exposing_registry.adoc

```diff
@@ -336,11 +354,14 @@ xref:access-logging-in-to-the-registry[logging in] to the secured and exposed
 registry, make sure you specify the registry in the login command. For example:
 
 ----
-docker login -e user@company.com -u f83j5h6 -p Ju1PeM47R0B92Lk3AZp-bWJSck2F7aGCiZ66aFGZrs2 registry.example.com:80
+# docker login -e user@company.com \
+    -u f83j5h6 \
+    -p Ju1PeM47R0B92Lk3AZp-bWJSck2F7aGCiZ66aFGZrs2 \
+    registry.example.com:80
 ----
 
 [[access-insecure-registry-by-exposing-route]]
-=== Exposing a Non-Secure Registry
+=== Manually Exposing a Non-Secure Registry
```

Comment:
Pull this up a level? It doesn't seem like it should be a subsection of the section on exposing a secure registry anymore.